### PR TITLE
Buttons to send new lexeme to lexeme-forms tool (in specified language)

### PR DESCRIPTION
**Pull request by arthurpsmith**
As promised :)

### CODE REVIEW

**Comment by fnielsen** [comment]
Where did my reviews go? Did you see them?

File: ordia/app/views.py [hunk 1]
```diff
@@ -388,7 +391,7 @@ def show_search():
                                    message=str(exception))
     else:
         search_results = []
-    return render_template("search.html",
+    return render_template("search.html", lang=lang,
```
Comment (fnielsen) [inline ordia/app/views.py]: Should perhaps be "language"

File: ordia/app/templates/search.html [hunk 2]
```diff
@@ -1,5 +1,18 @@
 {% extends "base.html" %}
 
+{% block scripts %}
+{{super()}}
+
+<script type="text/javascript" src="{{ url_for('static', filename='lexemeforms.js') }}"></script>
+<script type="text/javascript">
+ $(document).ready(function() {
+     lexemeTemplateSelection("{{ lang }}", "{{ query }}", "#lexeme-templates");
```
Comment (fnielsen) [inline ordia/app/templates/search.html]: The escape is not proper.

Comment (fnielsen) [reply ordia/app/templates/search.html]: https://stackoverflow.com/questions/12339806/escape-strings-for-javascript-using-jinja2

File: ordia/app/static/lexemeforms.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+function lexemeTemplateSelection(lang, lexeme, element) {
+    var templates_api_url = "https://tools.wmflabs.org/lexeme-forms/api/v1/template/";
+    var templates_prefix = "https://tools.wmflabs.org/lexeme-forms/template/";
+    
+    $.getJSON(templates_api_url, function(response) {
+        var templates_by_lang = lexemeTemplatesByLang(response);
+        var use_templates = templates_by_lang[lang];
+        for (var key in use_templates) {
+            $(element).append('<a href="' + templates_prefix + key + '?form_representation=' + lexeme + '"><button type="button" class="btn btn-secondary">' + use_templates[key] + '</button></a>');
```
Comment (fnielsen) [inline ordia/app/static/lexemeforms.js]: Spaces between the buttons are missing

**Comment by arthurpsmith** [comment]
I see reviews now - anyway thanks!

**Comment by fnielsen** [comment]
@arthurpsmith Thanks for the patch. It is now running, see, e.g.: http://127.0.0.1:5000/search?language=en&q=undeclared

**Comment by fnielsen** [comment]
> I see reviews now - anyway thanks!

Yes, I made some changes myself.

**Pull request merged**